templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.mm

```diff
@@ -83,6 +83,24 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull N
   return [super application:application continueUserActivity:userActivity restorationHandler:restorationHandler] || result;
 }
 
+// Leaves remote notification delegates for firebase ios sdk swizzling
```

Comment:
```suggestion
// Explicitly define remote notification delegates to ensure compatibility with some third-party libraries
```

maybe this?

Reply:
makes sense to me. i've updated the comments accordingly.